rules-reviewed/technology-usage/javaee-technology-usage.windup.xml

```diff
@@ -242,7 +242,146 @@
+                </graph-query>
+            </when>
+            <perform>
+                <technology-identified name="Java EE JACC">
+                    <tag name="Security"/>
+                    <tag name="Sustain"/>
+                    <tag name="Java EE"/>
+                </technology-identified>
+            </perform>
+        </rule>        
+        <rule id="javaee-technology-usage-00150">
+            <when>
+                <graph-query discriminator="TechnologyTagModel">
+                    <property name="name">MEJB</property>
+                </graph-query>
+            </when>
+            <perform>
+                <technology-identified name="Java EE MEJB">
+                    <tag name="Connect"/>
+                    <tag name="EJB"/>
+                    <tag name="Java EE"/>
+                </technology-identified>
+            </perform>
+        </rule>
+        <rule id="javaee-technology-usage-00160">
+            <when>
+                <graph-query discriminator="TechnologyTagModel">
+                    <property name="name">EAR</property>
+                </graph-query>
+            </when>
+            <perform>
+                <technology-identified name="Java EE Deployment">
+                    <tag name="Other"/>
+                    <tag name="Connect"/>
+                    <tag name="Java EE"/>
+                </technology-identified>
+            </perform>
+        </rule>
+        <rule id="javaee-technology-usage-00170">
+            <when>
+                <graph-query discriminator="TechnologyTagModel">
+                    <property name="name">WS Metadata</property>
+                </graph-query>
+            </when>
+            <perform>
+                <technology-identified name="Java EE WebServices Metadata">
+                    <tag name="Connect"/>
+                    <tag name="Http"/>
+                    <tag name="Java EE"/>
+                </technology-identified>
+            </perform>
+        </rule>        
+        <rule id="javaee-technology-usage-00180">
+            <when>
+                <graph-query discriminator="TechnologyTagModel">
+                    <property name="name">Common Annotations</property>
+                </graph-query>
+            </when>
+            <perform>
+                <technology-identified name="Common Annotations">
+                    <tag name="Connect"/>
+                    <tag name="EJB"/>
+                    <tag name="Java EE"/>
+                </technology-identified>
+            </perform>
+        </rule>
+        <rule id="javaee-technology-usage-00190">
+            <when>
+                <graph-query discriminator="TechnologyTagModel">
+                    <property name="name">JAXB</property>
+                </graph-query>
+            </when>
+            <perform>
+                <technology-identified name="Java EE JAXB">
+                    <tag name="Connect"/>
+                    <tag name="Other"/>
```

Comment:
Hi @jonathanvila, it seems like this value didn't change in the last commit.
Could you change it so that we will have both `JAXB` and `JSON-B` in the `Binding` box please?